src/tests-rosetta/api.ts

```diff
@@ -3289,6 +3289,226 @@ describe('Rosetta API', () => {
     );
   })
 
+  test('block/transaction coinbase', async () => {
+    const dbBlock = await db.getCurrentBlock();
+    const blockRes = dbBlock.result ?? undefined;
+    const block: DbBlock = {
+      block_hash: '0x1234',
+      index_block_hash: '0xdeadbeef',
+      parent_index_block_hash: blockRes ? blockRes.index_block_hash : '0x00',
+      parent_block_hash: blockRes ? blockRes.block_hash : "0x00",
+      parent_microblock_hash: '',
+      parent_microblock_sequence: 0,
+      block_height: blockRes ? blockRes.block_height + 1 : 1,
+      burn_block_time: 94869286,
+      burn_block_hash: '0x1234',
+      burn_block_height: 123,
+      miner_txid: '0x4321',
+      canonical: true,
+      execution_cost_read_count: 0,
+      execution_cost_read_length: 0,
+      execution_cost_runtime: 0,
+      execution_cost_write_count: 0,
+      execution_cost_write_length: 0,
+    };
```

Comment:
Could you build these blocks and transactions using `TestBlockBuilder`? You'll need to add a helper function for miner rewards (it will be very useful in the future too).